frontend/src/components/log-in/Login.jsx

```diff
@@ -9,12 +9,15 @@ import { useEffect } from "react";
 import { useLocation } from "react-router-dom";
 
 let LoginForm = null;
+let UnstractAndLlmWhispererLogo;
 let PRODUCT_NAMES = {};
 let SELECTED_PRODUCT;
 let selectedProductStore;
 let setSelectedProduct;
 try {
   LoginForm = require("../../plugins/login-form/LoginForm").LoginForm;
+  UnstractAndLlmWhispererLogo =
```

Comment:
@tahierhussain better to have some generic variable name like `CombinedLogo` instead of `UnstractAndLlmWhispererLogo` if possible

Reply:
Done, @jaseemjaskp